notification.c

```diff
@@ -133,6 +135,17 @@ struct mako_notification *get_notification(struct mako_state *state,
 	return NULL;
 }
 
+struct mako_notification *get_tagged_notification(struct mako_state *state,
+		const char *tag) {
+	struct mako_notification *notif;
+	wl_list_for_each(notif, &state->notifications, link) {
+		if (notif->tag && strcmp(notif->tag, tag) == 0) {
```

Comment:
Should we try to match by `app_id` as well? ie. the tags are namespaced per-app?

It doesn't seem like dunst does that, so maybe we shouldn't. If we were to standardize this capability, I think this would be desirable to avoid tag collisions.

Reply:
I think it makes sense too. Most scripts probably won't bother setting the app ID anyway, but every little helps. I'll add it for now.